tests/int/test-just_install_functions.bsh

```diff
@@ -23,8 +23,10 @@ begin_test "cmake-install"
 (
   setup_test
 
+  # redirect mktemp commands to $TESTDIR (does not work on a mac)
+  [ "${VSI_OS}" != "darwin" ] && export TMPDIR="${TESTDIR}"
```

Comment:
This is inside a test, won't it return false and fail if not darwin?

Make it a normal if statement

Reply:
If that’s true, why did the test succeed in appveyor?

Reply:
Ah, because `( set -eu; false && false; echo hi; )` apparently always echo's hi, must be one of those corner cases `set -e` is ignored